### PR TITLE
Add tables => displya method, emojis

### PR DESCRIPTION
**Pull request by TkachenkoKaterina**

### CODE REVIEW

File: helpers/emojis.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def smiley_face():
+    return "😊"
+
+def thumbs_up():
+    return "👍"
+
+def check_mark():
+    return "✔️"
```
Comment (Iryna-Holova) [inline helpers/emojis.py]: use constants indtead functions

File: helpers/display.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    table = [[
+        str(contact.name),
+        phones_str,
+        str(contact.birthday) if contact.birthday else "N/A",
+        str(contact.address) if contact.address else "N/A"
+    ]]
+
+    table_str = tabulate(table, headers=colored_headers, tablefmt="grid")
+
+    return table_str
```
Comment (Iryna-Holova) [inline helpers/display.py:40]: single function for both contact and contacts would be better

**Review by Iryna-Holova — CHANGES_REQUESTED** [review]
comments in files

File: helpers/emojis.py (hunk 1 above)
Comment (TkachenkoKaterina) [reply helpers/emojis.py]: Deleted

File: helpers/display.py (hunk 2 above)
Comment (TkachenkoKaterina) [reply helpers/display.py:40]: Later come back to this question

**Review by Iryna-Holova — APPROVED** [review]

**Pull request merged**